src/main/java/frc/robot/Robot.java

```diff
@@ -39,18 +40,28 @@
     private SpinnyBoiSubsystem spinnyBoiSubsystem;
     private TurretSubsystem turretSubsystem;
     private BallManagementSubsystem ballManagementSubsystem;
-    private VisionSubsystem visionSubsystem;
-    
+    private Vision vision;
+
     private ExecutorService smartDashboardThread = Executors.newSingleThreadExecutor();
 
     private boolean disableDash = false;
 
     @Override
     public void robotInit() {
 
+
+        climberSubsystem = new ClimberSubsystem();
+        climberSubsystem.initialize();
```

Comment:
We already init the climber subsystem below. in a loop.

Reply:
oh, i just had to move it since on my local copy it was in a weird spot